test/spmd/test_xla_sharding.py

```diff
@@ -164,6 +164,22 @@ def test_transfer_sharded_data_to_host(self):
     t1 = xt1.cpu()
     self.assertTrue(torch.allclose(t1, torch.ones(16, 16)))
 
+  def test_send_cpu_data_to_device_with_sharding(self):
+    tensor = torch.arange(16).reshape(4, 4)
+    mesh = self._get_mesh((1, self.n_devices))
+    sharding_spec = xs.ShardingSpec(mesh, (0, 1))
+    self.assertTrue(sharding_spec.can_apply(tensor))
+    xtensors = xm.send_cpu_data_to_device([tensor],
+                                          xm.xla_device(),
```

Comment:
so... the device here will be a virtual device? because `XLA_USE_SPMD=1`?

Reply:
This would be `xla` device in Python, and all `xla` devices will be understood as a virtual `SPMD:0` device, until the actual materialization. Unless @steventk-g already worked on the virtual device Python interface, we should take this change separately, to expose a Python level virtual device.